plugins/bookmarks.js

```diff
@@ -992,14 +992,16 @@ window.plugin.bookmarks.loadStorageBox = function() {
       if(fullUpdated) {
         window.plugin.bookmarks.refreshBkmrks();
         window.plugin.bookmarks.resetAllStars();
-        console.log('BOOKMARKS: synchronized all after offline');
+        window.runHooks('pluginBkmrksSyncEnd', {"target": "all", "action": "sync"});
+        console.log('BOOKMARKS: synchronized all from drive after offline');
         return;
       }
 
       if(!e) return;
       if(e.isLocal) {
         // Update pushed successfully, remove it from updatingQueue
         delete window.plugin.bookmarks.updatingQueue[e.property];
+        console.log('BOOKMARKS: synchronized to drive');
```

Comment:
And this

Reply:
console.log: me I like it , you do not I guess.

Reply:
I like it, when it needed.
But I do not like when I see a mess of console.log (from dozens sources) and unable to find that one I need.
Just install old iitc-me and see what I mean.
And I've put much effort to make log useful again in IITC-CE.